### PR TITLE
[New Model] Ess supplier request1.0.0

### LINKED ISSUE #57: [New Model]: (Traceability) ESS Supplier Request
## Model Description

This model describes the schema of a notification to be provided by an inquiring company to his supply chain partners. The notification requests a supply chain analysis on recipient side with the objective to analyze, if a given company (as specified in a given environmental of social standards (ESS) incident) is part of the supply chain of the inquiring company.

## Relevant Standards

Model is based on Catena-X Quality Notifications, which is the de-facto standard for notifications in CX

## Example Data

{
"header": {
"notificationId": "e0f8c1b9-ba5a-4b7e-a431-b2d03d129705",
"senderBpn": "BPNL00000003BW3S",
"recipientBpn": "BPNL00000003BV4H",
"senderEdc": "https://edc.company-xyz.io/BPNL00000003BW3S",
"replyAssetId": "ess-response-asset",
"replyAssetSubPath" : "optionalSubPath",
"notificationType": "ess-supplier-request"

},
"content": {
    "incidentBpn" : "BPNS123456"
    "concernedCatenaXIds" : ["CxUUID"]
}
}

## MS1 Criteria

- [x] The proposed aspect model does not exist already
- [x] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [x] The proposal references a Catena-X use case
- [x] Relevant standards are mentioned/linked
- [x] A first draft of the data structure is provided in an example

### PR DESCRIPTION
**Pull request by sschul33**
## Description

 -->

Closes #57 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "name" and "description"** in English language. 
- [x] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [ ] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [ ] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [ ] All required reviewers have approved this PR (see reviewers section)
- [ ] The new aspect (version) will be implemented by at least one data provider
- [ ] The new aspect (version) will be consumed by at least one data consumer
- [ ] There exists valid test data
- [ ] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [ ] The model has at least version '1.0.0'

### CODE REVIEW

**Comment by sschul33** [comment]
@bs-jokri : Please review

@BenediktMuellerIML , @jacewski-bosch : fyi

**Comment by sschul33** [comment]
Modeling_Team

**Comment by bs-jokri** [comment]
Note: The model requires the shared aspect io.catenax.notification_header/1.0.0/NotificationHeader.ttl in order to validate. This shared aspect is provided in another PR, which needs to be merged before.
https://github.com/eclipse-tractusx/sldt-semantic-models/pull/128/files

File: io.catenax.ess_supplier_request/1.0.0/EssSupplierRequest.ttl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    bamm:characteristic :EssSupplierRequestContentCharacteristic.
+:EssSupplierRequestContentCharacteristic a bamm:Characteristic;
+    bamm:preferredName "EssSupplierRequestContentCharacteristic"@en;
+    bamm:dataType :EssSupplierRequestContentEntity.
+:incidentBpn a bamm:Property;
+    bamm:preferredName "incidentBpn"@en;
+    bamm:description "The BPNL or BPNS of the accused company of the ESS incident.\nThe receipient has to evaluate, if the incidentBPN is a (sub-) supplier of the given list of parts."@en;
+    bamm:characteristic :IncidentBpnCharacteristic.
+:IncidentBpnCharacteristic a bamm:Characteristic;
+    bamm:preferredName "IncidentBpnCharacteristic"@en;
```
Comment (bs-jokri) [inline io.catenax.ess_supplier_request/1.0.0/EssSupplierRequest.ttl:54]: Same here. Please adjust all preferred names to be written in standard English (no camel case)

File: io.catenax.ess_supplier_request/1.0.0/EssSupplierRequest.ttl [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    bamm:dataType xsd:string.
+:concernedCatenaXIds a bamm:Property;
+    bamm:preferredName "concernedCatenaXIds"@en;
+    bamm:description "A list of Catena-X IDs. The Catena-X Ids refer to the parts/components which are supplied by the recipient of the ESS supplier request to the sender and for which the supply chain analysis shall be performed by the recipient."@en;
```
Comment (bs-jokri) [inline io.catenax.ess_supplier_request/1.0.0/EssSupplierRequest.ttl:58]: ID Id -> be consistent

**Pull request closed without merging**